src/App.tsx

```
@@ -1,7 +1,26 @@
+import { useEffect } from "react";
 import { RouterProvider } from "react-router-dom";
+import { useAttendanceCheck } from "./hooks/attendance/useAttendanceCheck";
 import router from "./routes/router";
 
 function App() {
+  const { handleCheckAttendance } = useAttendanceCheck();
+
+  useEffect(() => {
```

Biome (2.3.13)

[error] This hook does not specify its dependency on handleCheckAttendance.

This dependency is being used here, but is not specified in the hook dependency list.

This dependency is being used here, but is not specified in the hook dependency list.

React relies on hook dependencies to determine when to re-compute Effects.
Failing to specify dependencies can result in Effects not updating correctly when state changes.
These "stale closures" are a common source of surprising bugs.
Either include it or remove the dependency array.
Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)